src/shader_compiler.h

```diff
@@ -30,7 +30,8 @@ namespace amber {
 class ShaderCompiler {
  public:
   ShaderCompiler();
-  explicit ShaderCompiler(const std::string& env);
+  explicit ShaderCompiler(const std::string& env,
```

Comment:
nit: explicit not needed now that there is more then 1 argument.

Reply:
Fixed